### ISSUE TITLE
Defining environment variables in .travis.yml

### ISSUE THREAD

**Issue by SamVerschueren** [vdemedes/trevor#20]
As documented [here](http://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-.travis.yml).

I would start simple and implement the basics first.

```
language: 'node_js'
env:
  - FOO=foo BAR=bar
  - FOO=bar BAR=foo
node_js:
  - 'stable'
  - '0.12'
```

- Node (stable) with `FOO=foo BAR=bar`
- Node (stable) with `FOO=bar BAR=foo`
- Node 0.12 with `FOO=foo BAR=bar`
- Node 0.12 with `FOO=bar BAR=foo`

Later on, things like the [matrix](http://docs.travis-ci.com/user/environment-variables/#Global-Variables) can be implemented.

**Comment by vdemedes** [comment]
This is a good suggestion indeed. The only reason I did not implement this yet, because it involves creating a matrix (which has to be tested really good).

I am thinking one of the next steps we need to take is to write tests, because things are getting interesting :)

**Comment by SamVerschueren** [comment]
Yes I was thinking in terms of testing as well. Definetely the next step, even before implementing the services part.

**Comment by sindresorhus** [comment]
Soon you can just start your own Travis-like SaaS, just better. Trevor CI ;)

**Comment by vdemedes** [comment]
Haha, I should!

**Comment by SamVerschueren** [comment]
Sure, why not :)

**Comment by cades** [comment]
:+1:
